### ISSUE TITLE
VH01: Bug with Change party member

### ISSUE THREAD

**Issue by Yuuyatails** [EasyRPG/Player#1633]
#### Describe the issue in detail and how to reproduce it:
For some reasons, when starting a new game with other character (i.e. Selena, the blonde-haired girl), the main party member (a.k.a the protagonist), for whatever reason, is still Nanako (The red-haired girl), with Selena being added as secondary party member instead. This bug also causes the 'Change the protagonist/allies' ('主人公・仲間の変更' in Japanese) functionality in the debug tool (Something that you get when start a new game in debug mode) to fail to work properly, as it fails to swap out the main party member.

One thing I noticed is that you can only trigger this bug while starting a new game with character other than Nanako; Party change (via debug tool) still work if you start a new game with Nanako.

This happens on both Windows and Android platform, as such, is it a cross-platform issue. Since I can only play the game on Android without the vfs bug (as seen in the Windows version of the player), there is no log.

**Comment by Ghabry** [comment]
Thanks for your detailed report, we will take a look.
The Windows file problem was just fixed btw.

**Comment by Ghabry** [comment]
@Yuuyatails 
I can't reproduce this bug. It is possible that the Android version (from the Play store) is too old and we fixed the bug in the meanwhile.
Please test again with the latest version of Player on Windows, as the filename bug is fixed now.

**Comment by Yuuyatails** [comment]
Sorry Ghabry, but I'm have to close this issue, as I found out this is not the bug that caused by the player, but rather it was caused by the game itself. 

To be specific, since version 181020 of the game, the party change function will bugged out if starting a new game with character other than Nanako. The workaround is using the debug tool to switch back to Nanako and then to other character.

Again, I'm sorry for taking your time.
